packages/api-clients/api-codegen-preset/README.md

```diff
@@ -169,6 +169,84 @@ export default {
 };
 ```
 
+#### Example `.graphqlrc.ts` file with to generate types for UI extensions
+You can specify multiple APIs in your `.graphqlrc.ts` file by adding multiple projects.
+
+```js
+import { LATEST_API_VERSION } from "@shopify/shopify-api";
+import { shopifyApiProject, ApiType } from "@shopify/api-codegen-preset";
+import type { IGraphQLConfig } from "graphql-config";
+
+function getConfig() {
+  const config: IGraphQLConfig = {
+    projects: {
+      default: shopifyApiProject({
+        apiType: ApiType.Storefront,
+        apiVersion: LATEST_API_VERSION,
+        documents: ["./extensions/**/*.{js,ts,jsx,tsx}", "./extensions/.server/**/*.{js,ts,jsx,tsx}"],
+        outputDir: "./extensions/types",
+      }),
+      UIExtensions: shopifyApiProject({
+        apiType: ApiType.Admin,
+        apiVersion: LATEST_API_VERSION,
+        documents: ["./app/**/*.{js,ts,jsx,tsx}", "./app/.server/**/*.{js,ts,jsx,tsx}" ],
+        outputDir: "./app/types",
+      }),
+    },
+  };
+
+const config = getConfig();
+export default config;
+```
+
+
+### Example `graphqlrc.ts` file for autocompletion for Shopify Function Extensions
+```ts
+import fs from "fs";
+import { LATEST_API_VERSION } from "@shopify/shopify-api";
+import { shopifyApiProject, ApiType } from "@shopify/api-codegen-preset";
+import type { IGraphQLConfig } from "graphql-config";
+
+function getConfig() {
+  const config: IGraphQLConfig = {
+    projects: {
+      default: shopifyApiProject({
+        apiType: ApiType.Storefront,
```

Comment:
This may be me who is confused, but Function Extensions are [these ones](https://shopify.dev/docs/apps/build/functions), right? If so, should be renamed to "... For Shopify Functions".

I don't think these has access to the Storefront API?

Reply:
Put up a [PR here](https://github.com/Shopify/shopify-app-js/pull/2208) to clarify.